### DISCUSSION TITLE
Something wrong with my PC ??

### DISCUSSION THREAD

**Discussion by CrewieAustin** [Monika-After-Story/MonikaModDev#7766] [QA - tech support + general] [answered]
I was visiting Monika today as usual, but instead of her normally showing up. A warning screen popped up telling me I have to upgrade my DirectX. Do I have to do anything here ?
![Monika_Error](https://user-images.githubusercontent.com/86544170/132815565-2091610d-80da-4b9f-8c17-3944db9f34c6.png)

**Comment by ConnorTippets** [comment, marked as answer]
If I understand your problem correctly, just wait for the installer to finish and then click Restart and MAS should be back to normal
